### DISCUSSION TITLE
修复网格吸附导致的偏移问题时遇到了一个纠结点😖

### DISCUSSION THREAD

**Discussion by DymoneLewis** [didi/LogicFlow#1970] [Polls]
因为印象中收到过一些因为网格吸附导致初始化渲染的节点和边对不齐的问题，所以决定最近集中修复一下，目前想到了两个修复思路：
1. 初始化时节点照常吸附到网格上，边自适应连接到对应的锚点
2. 初始化时节点不做处理，在用户操作时再进行网格吸附
现在在具体往哪个方向上走比较纠结，想看看各位朋友的想法

**Comment by DymoneLewis** [comment]
其他想法欢迎在评论里留言

**Comment by boyongjiong** [comment]
「边自适应连接到对应的锚点」 怎么理解这句话
